indico_themes_canonical/static/css/ubuntu-summit/registration.css

```diff
@@ -97,7 +97,35 @@ i.icon.icon-phone {
   width: 100%;
 }
 
-/* Styles for the form */
+/* Date input */
+.SingleDatePicker .SingleDatePickerInput__withBorder {
+  border: none;
+}
+
+button.SingleDatePickerInput_calendarIcon {
+  padding: 0;
+}
+
+/* Yes/No */
+
+.events-registration-client-styles-___regform-module__boolean-field___0geJs .ui.button {
+  background: var(--brand-off-white);
```

Comment:
Can we set both a `background` and a `color` here? This makes it more explicit and avoids adverse effects where the `color` might be white as well in a different context.

Reply:
Good catch, just updated it